templates/base_index.html

```diff
@@ -1104,7 +1127,41 @@ <h2 class="p-heading--1 u-no-margin--bottom">Multi-cloud applications&nbsp;&ndas
       {# include "ubuntu/takeovers/_footer_homepage.html" #}
     {% endblock footer_content %}
 
-    <script>
+    <script type="module">
+      const testTakeover = document.getElementById('test-takeover');
+      const mainTakeover = document.getElementById('takeover');
+
+      const getCookie = () => document.cookie.match(new RegExp('(^| )' + "control_or_variant" + '=([^;]+)'));
+      // Switch for turning takeover switching on and off on demand
+      const takeoverSwitch = true;
+      // check if user doesn't already have a group
+      if (!getCookie()) {
```

Comment:
I have a question. From looking at this condition it seems like the "takeoverSwitch" will only work for people who don't have the "control_or_variant" cookie. What about those who have the cookie and we at some point we decide we do not want to split people into two different groups and show them different layouts?